overdue/src/test/resources/org/killbill/billing/overdue/BadOverdue.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Again, I would suggest renaming this to something like `OverdueWithTotalUnpaidInvoiceBalanceCondition`